homeassistant/components/stream/worker.py

```diff
@@ -278,7 +281,7 @@ def peek_first_dts() -> bool:
                     first_packet = packet
                     initial_packets.append(packet)
             # Get first_dts from subsequent frame to first keyframe
-            while segment_start_dts is None or (
+            while segment_start_dts is None or (  # type: ignore
```

Comment:
Ignoring this line and L327 below. They seem to be spurious

Reply:
It's not, add type hint to `first_packet`:
```python
first_packet: av.Packet | None = None
```
```suggestion
            while segment_start_dts is None or (
```